Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -355,19 +353,14 @@ def organize_intervals(intervals):
             if len(e) <= len(s):
                 e = s[:-len(e)] + e
                 if s <= e:
-                    ni1 = []
-                    for i1 in range(int(s), int(e) + 1):
-                        ni1.append(str(i1))
+                    ni1 = [str(i1) for i1 in range(int(s), int(e) + 1)]
                     maxlen = len(max(ni1, key=lambda num: len(num)))
                     if len(min(ni1, key=lambda num: len(num))) != maxlen:
                         # TO DO: improve this.
                         ni2 = []
                         for i1 in ni1:
-                            add_zeros = maxlen - len(i1)
-                            i2 = ""
-                            for j in range(0, add_zeros):
-                                i2 += "0"
-                            i2 += i1
+                            i2 = "".join(["0" for j in
+                                          range(0, maxlen - len(i1))]) + i1
```

Comment:
Wouldn\`t `"0" * (maxlen - len(i1)) + i1` be more readable?
```
>>> "".join("0" for i in range(0,3))
'000'
>>> "0" * 3
'000'
```

Reply:
I wasn't aware that such multiplication is possible. True, your suggestion looks better, will do.